otter/tap/api.py

```diff
@@ -275,10 +276,12 @@ def on_client_ready(_):
                 stop=partial(call_after_supervisor,
                              kz_client.stop, supervisor)))
 
-            # setup converger service
-            converger_service = Converger(reactor, kz_client, dispatcher)
+            # setup ConvergenceStarter service
+            converger_service = ConvergenceStarter(dispatcher)
```

Comment:
Non-blocking: It looks like `ConvergenceStarter` need not be a service. Can be a class that encapsulates `dispatcher`. In fact, I wonder if its possible to have `start_convergence` as a function that takes `dispatcher` as first argument and this code just `partial`s it with dispatcher.